packages/odyssey-lint-staged/src/index.js

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+module.exports = {
+  "*": "yarn prettier --write . --ignore-path .gitignore",
+  ".scss": "stylelint",
+  ".js,.jsx,.ts,.tsx": "eslint .",
+};
```

Comment:
\o/

Reply:
See my note about using lint-staged in CI rather than the top level explicit calls to eslint and stylelint

Reply:
We spoke about this over zoom, determined that we would not be doing this as it is not a use-case lint-staged supports!